Doc/library/unittest.rst

```diff
@@ -1745,6 +1764,21 @@ Loading and running tests
 
       This affects all the :meth:`loadTestsFrom\*` methods.
 
+   .. attribute:: testNamePatterns
+
+      List of Unix shell-style wildcard test name patterns that test methods
+      have to match to be included in test suites (see ``-v`` option).
+
+      If this attribute is not `None` (the default), all test methods to be
```

Comment:
The sphinx build says:
`[2] library/unittest.rst:1772: default role used`
It points to line 1772 of unittest.rst.
I think it's because single backticks was used around None.
Try using double backticks instead.